plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesPluginImpl.kt

```diff
@@ -1199,10 +1187,33 @@ class GesturesPluginImpl : GesturesPlugin, GesturesSettingsBase {
         (velocityXY / 7.0 / pitchFactor + ANIMATION_DURATION_FLING_BASE).toLong()
 
       if (mapTransformDelegate.terrainEnabled()) {
-        mapTransformDelegate.drag(
+        mapTransformDelegate.dragStart(
```

Comment:
@mpulkki-mapbox @astojilj - here and below is actual part how platform dragging looks like.